src/transaction/combine.ts

```diff
@@ -1,42 +1,80 @@
 import * as _ from 'lodash'
 import binary from 'ripple-binary-codec'
-import * as utils from './utils'
 import BigNumber from 'bignumber.js'
+import { ValidationError } from '../common/errors'
```

Comment:
Can we adjust the spacing on these imports to be `{ValidationError}` to match the rest of the code. This should get fixed when we address linting, though.